io/net/Network_config.py

```diff
@@ -110,7 +110,8 @@ def test_networkconfig(self):
                     self.fail("mismatch in mtu")
             if 'MTU' in val:
                 cmd, mtu_val = val.split('MTU:')
-                mtu_val, cmd = mtu_val.split(' ')
+                mtu_val, cmd = mtu_val.split(' M')
```

Comment:
The script would error out at this line: mtu_val, cmd = mtu_val.split(' ') 
with the error "ERROR| ValueError: too many values to unpack" as it was retriving the MTU vaue as "1500  Metric:1"

In Rhel and Ubuntu, the 'mtu' value is displayed as "mtu 1500" 
In Sles, it is seen as "MTU:1500  Metric:1" 

So when Iterating in the list and finding the mtu value, With the patch, the value before ' M' where 'M is Metric, is put in mtu_val which is the value we need.
cmd, mtu_val = val.split('MTU:')
mtu_val, cmd = mtu_val.split(' M')
mtu_val = mtu_val.strip()